### DISCUSSION TITLE
Feature request: Allow bookmark reordering

### DISCUSSION THREAD

**Discussion by lubokkanev** [PaulWoitaschek/Voice#3168] [Ideas]
### Overview

Sometimes I forget to add a bookmark and I go back through the book to add it. In that case the bookmark gets on top of the others, but it's about an earlier time. I'd like to be able to reorder it. 

### How to reproduce

1. Create two bookmarks.
2. Try to change their order. 

### Version

8.1.3

### Android Version

33

### Device Model

SM-G980F

**Comment by luzpaz** [comment]
This would be amazing if implemented. When making bookmarks, sometimes certain nuances are missed when listening back. When you go to make a new bookmark and save it...it is now **so** out of of context chronologically that it's easy to lose in an audiobook that you've made over at least a dozen bookmarks for. So this feature would be **very** helpful in that use-case. 

I imagine I could click and hold on the bookmark then drag it up or down to the place I want to drop in the list order.  

Note: Currently on v25.10.5 at the time of writing this comment.
